pkg/apk/apk/implementation.go

```diff
@@ -524,9 +524,9 @@ func (a *APK) CalculateWorld(ctx context.Context, allpkgs []*RepositoryPackage)
 	resolved := make([]*APKResolved, len(allpkgs))
 
 	// A slice of pseudo-promises that get closed when expanded[i] is ready.
-	done := make([]chan struct{}, len(allpkgs))
+	done := make([]chan error, len(allpkgs))
```

Comment:
We can actually just delete these because there's nothing that reads from these channels. I think this was a copy-paste leftover.